mix.exs

```diff
@@ -47,6 +47,7 @@ defmodule LambdaEthereumConsensus.MixProject do
       {:telemetry_poller, "~> 1.0"},
       {:telemetry_metrics, "~> 0.6"},
       {:telemetry_metrics_prometheus, "~> 1.1.0"},
+      {:ex2ms, "~> 1.6", runtime: false},
```

Comment:
what is this for?

Reply:
Erlang had a kind-of-macro system that was used by the `ets` module for the [`fun2ms/1` "function"](https://www.erlang.org/doc/man/ets#fun2ms-1). This is the Elixir equivalent.